Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -608,6 +609,10 @@ final class StatusTableViewController: LoopChartsTableViewController {
             statusRowMode = .enactingBolus
         } else if case .canceling = bolusState {
             statusRowMode = .cancelingBolus
+        } else if case .suspended = basalDeliveryState {
+            statusRowMode = .pumpSuspended(resuming: false)
+        } else if self.basalDeliveryState == .resuming {
```

Comment:
[nit] Why the different comparison for this line and the line 2 above? Suggest changing this line to:
```
} else if case .resuming = basalDeliveryState {
```